test/apps/multi-viewport/app.js

```diff
@@ -52,33 +50,19 @@ const DEFAULT_VIEWPORT_PROPS = {
   up: [0, 0, 1] // Defines up direction, default positive y axis
 };
 
-const transitionInterpolator = new LinearInterpolator([
-  'longitude',
-  'latitude',
-  'zoom',
-  'bearing',
-  'pitch',
-  'position'
-]);
-
 class Root extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      fov: 50,
-      animatePosition: true,
-      animateBearing: true,
-
-      viewportProps: {
-        ...DEFAULT_VIEWPORT_PROPS,
-        width: 500,
-        height: 500
+      viewState: {
+        '1st-person': {...DEFAULT_VIEWPORT_PROPS},
```

Comment:
This allows us to have multiple separate viewStates but not really share view states so easily?

One design I had in mind had the `Views` specifying which viewState id they wanted. But that approach has its issues too.